### ISSUE TITLE
NoClassDefFoundError: org/bukkit/craftbukkit/v1_21_R1/CraftChunk

### ISSUE THREAD

**Issue by aematsubara** [olijeffers0n/PatheticAPI#108]
### Stack trace

```
[12:51:06] [Server thread/ERROR]: Error occurred while enabling VehiclesWASD v1.2.3 (Is it up to date?)
java.lang.NoClassDefFoundError: org/bukkit/craftbukkit/v1_21_R1/CraftChunk
	at me.matsubara.vehicles.util.pathetic.provider.v1_21_R1.v1_21_R1ChunkDataProviderImpl.<clinit>(v1_21_R1ChunkDataProviderImpl.java:22) ~[?:?]
	at me.matsubara.vehicles.util.pathetic.provider.ChunkDataProviderResolver.determineChunkDataProvider(ChunkDataProviderResolver.java:50) ~[?:?]
	at me.matsubara.vehicles.util.pathetic.provider.ChunkDataProviderResolver.<init>(ChunkDataProviderResolver.java:35) ~[?:?]
	at me.matsubara.vehicles.util.pathetic.model.snapshot.FailingSnapshotManager.<clinit>(FailingSnapshotManager.java:47) ~[?:?]
	at me.matsubara.vehicles.util.pathetic.model.pathing.pathfinder.AbstractPathfinder.<clinit>(AbstractPathfinder.java:55) ~[?:?]
	at me.matsubara.vehicles.util.pathetic.mapping.PatheticMapper.newPathfinder(PatheticMapper.java:53) ~[?:?]
	at me.matsubara.vehicles.command.VehiclesCommands.<init>(VehiclesCommands.java:84) ~[?:?]
	at me.matsubara.vehicles.VehiclesPlugin.onEnable(VehiclesPlugin.java:172) ~[?:?]
	at org.bukkit.plugin.java.JavaPlugin.setEnabled(JavaPlugin.java:267) ~[spigot-api-1.21.3-R0.1-SNAPSHOT.jar:?]
	at org.bukkit.plugin.java.JavaPluginLoader.enablePlugin(JavaPluginLoader.java:342) ~[spigot-api-1.21.3-R0.1-SNAPSHOT.jar:?]
	at org.bukkit.plugin.SimplePluginManager.enablePlugin(SimplePluginManager.java:492) ~[spigot-api-1.21.3-R0.1-SNAPSHOT.jar:?]
	at org.bukkit.craftbukkit.v1_21_R2.CraftServer.enablePlugin(CraftServer.java:579) ~[spigot-1.21.3-R0.1-SNAPSHOT.jar:4358-Spigot-5eb8a94-6bb0db5]
	at org.bukkit.craftbukkit.v1_21_R2.CraftServer.enablePlugins(CraftServer.java:493) ~[spigot-1.21.3-R0.1-SNAPSHOT.jar:4358-Spigot-5eb8a94-6bb0db5]
	at net.minecraft.server.dedicated.DedicatedServer.e(DedicatedServer.java:236) ~[spigot-1.21.3-R0.1-SNAPSHOT.jar:4358-Spigot-5eb8a94-6bb0db5]
	at net.minecraft.server.MinecraftServer.y(MinecraftServer.java:1032) ~[spigot-1.21.3-R0.1-SNAPSHOT.jar:4358-Spigot-5eb8a94-6bb0db5]
	at net.minecraft.server.MinecraftServer.lambda$spin$0(MinecraftServer.java:329) ~[spigot-1.21.3-R0.1-SNAPSHOT.jar:4358-Spigot-5eb8a94-6bb0db5]
	at java.base/java.lang.Thread.run(Thread.java:1583) [?:?]
Caused by: java.lang.ClassNotFoundException: org.bukkit.craftbukkit.v1_21_R1.CraftChunk
	at org.bukkit.plugin.java.PluginClassLoader.loadClass0(PluginClassLoader.java:160) ~[spigot-api-1.21.3-R0.1-SNAPSHOT.jar:?]
	at org.bukkit.plugin.java.PluginClassLoader.loadClass(PluginClassLoader.java:112) ~[spigot-api-1.21.3-R0.1-SNAPSHOT.jar:?]
	at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:526) ~[?:?]
	... 17 more
```

### Actions to reproduce (if known)

1. Try to create a org.patheloper.api.pathing.Pathfinder with Pathfinder#newPathfinder(PathfinderConfiguration):
In my case:
```
this.pathfinder = PatheticMapper.newPathfinder(PathfinderConfiguration.createAsyncConfiguration()
                .withAllowingFailFast(true)
                .withAllowingFallback(true)
                .withAllowingDiagonal(true)
                .withLoadingChunks(true)
                .withMaxIterations(24000)
                .withAsync(true));
```
2. Turn on the server (Spigot 1.21.3). The error appears when trying to initialize the Pathfinder

### Other

Using: com.github.patheloper.pathetic:pathetic-mapping:3.2.2

It seems to be a problem only with Spigot 1.21.3 as earlier versions work without problems.

**Comment by bsommerfeld** [comment]
hi, thanks for reporting. will investigate ASAP

**Comment by bsommerfeld** [comment]
this issue is fixed and backported to version [3.2.2](https://github.com/patheloper/pathetic/releases/tag/3.2.2)
